setup.py

```diff
@@ -23,8 +23,8 @@
         "microcosm-logging>=1.5.0",
         "psycopg2-binary>=2.7.5",
         "pytz>=2018.5",
-        "SQLAlchemy>=1.2.11",
-        "SQLAlchemy-Utils>=0.33.3",
+        "SQLAlchemy>=1.4.10",
+        "SQLAlchemy-Utils>=0.37.0",
```

Comment:
SQLALchemy-utils released this recently with support for sqlalchemy 1.4.x

see https://github.com/kvesteri/sqlalchemy-utils/commit/fd85312da82adcaaeb98a37c91f75c094219033f#diff-ff3c479edefad986d2fe6fe7ead575a46b086e3bbcf0ccc86d85efc4a4c63c79R10

cc/ @german-glob @piercefreeman